Makefile

```diff
@@ -371,13 +371,13 @@ CHECK		= sparse
 
 # Use the wrapper for the compiler.  This wrapper scans for new
 # warnings and causes the build to stop upon encountering them.
-CC		= $(srctree)/scripts/gcc-wrapper.py $(REAL_CC)
+# CC		= $(srctree)/scripts/gcc-wrapper.py $(REAL_CC)
 
 CHECKFLAGS     := -D__linux__ -Dlinux -D__STDC__ -Dunix -D__unix__ \
 		  -Wbitwise -Wno-return-void $(CF)
 CFLAGS_MODULE   =
 AFLAGS_MODULE   =
-LDFLAGS_MODULE  =
+LDFLAGS_MODULE  = --strip-debug
```

Comment:
That's fine, but what if you wanted the debug symbols?  :)

And it has nothing to do with the original goal of this patch, so please strip it out and submit it separately, so I can reject it there :)  These options can be changed on the command line if you really want it, don't go hard-coding platform specific things into the makefile, that's not ok.